src/client/components/dragAndDrop/drag-and-drop.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Pizzicato from 'pizzicato';
+import Dropzone from 'react-dropzone';
+import { createAudioData, playPauseSoundFromFile } from '../../actions/audioActions';
+
+import './drag-and-drop.css';
```

Comment:
Change name for of css file, like dragAndDrop.css

Reply:
And seems problem you have with naming of js file

Reply:
done